esmvaltool/cmorizers/obs/cmorize_obs_eraintland.py

```diff
@@ -0,0 +1,132 @@
+"""ESMValTool CMORizer for ERA-Interim-Land data.
```

Comment:
I think this file is no longer needed, right?

Reply:
Correct, I will take it out.